BubblesKotlin/app/src/main/java/com/example/android/bubbles/ui/chat/ChatFragment.kt

```diff
@@ -47,13 +48,16 @@ class ChatFragment : Fragment() {
     companion object {
         private const val ARG_ID = "id"
         private const val ARG_FOREGROUND = "foreground"
-
-        fun newInstance(id: Long, foreground: Boolean) = ChatFragment().apply {
-            arguments = Bundle().apply {
-                putLong(ARG_ID, id)
-                putBoolean(ARG_FOREGROUND, foreground)
+        private const val ARG_PREPOPULATE_TEXT = "prepopulate_text"
+
+        fun newInstance(id: Long, prepopulateText: String?, foreground: Boolean) =
```

Comment:
Add default param values? (so you can stop explicitly passing `null` above for `prepopulateText`

Reply:
Done.